Tests/NIOPosixTests/SelectorTest.swift

```diff
@@ -122,7 +123,7 @@ class SelectorTest: XCTestCase {
     func testWeDoNotDeliverEventsForPreviouslyClosedChannels() {
         /// We use this class to box mutable values, generally in this test anything boxed should only be read/written
         /// on the event loop `el`.
-        class Box<T> {
+        final class Box<T>: @unchecked Sendable {
```

Comment:
This sendable annotation is definitely a lie. It should probably have been replaced by something like NIOLoopBoundBox or NIOLockedValueBox.

Reply:
I'll fix this up